errors/error.go

```diff
@@ -252,7 +252,13 @@ func (e *Error) error(verbose bool) string {
 			}
 		case error:
 			if v != nil {
-				errParts = append(errParts, v.Error())
+				errmsg := ""
+				if e, ok := v.(interface{ Detailed() string }); ok && verbose {
```

Comment:
Important to debug an error.Error that has a list inside it, so we can see the list in detail too